controllers/evalhub/metrics.go

```
@@ -0,0 +1,117 @@
+
+import (
+	"strings"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+const (
+	resultSuccess = "success"
```

golangci-lint (2.12.2)

[error] const resultSuccess is unused

(unused)